test/hotspot/jtreg/runtime/cds/appcds/SharedArchiveConsistency.java

```diff
@@ -89,261 +62,21 @@
     public static int num_regions = shared_region_name.length;
     public static String[] matchMessages = {
         "Unable to use shared archive",
+        "The shared archive file has a bad magic number",
+        "Unable to map shared spaces",
```

Comment:
Why are these new errors needed? Did you change any behavior of the tests?

Reply:
Thanks for mentioning this. There is an issue to use FileChannel:
            outputChannel.transferFrom(inputChannel, 0, offset);
         + outputChannel.position(offset);    // <---- need explicitly positioned or the following write to overwrite from offset 0.
            outputChannel.write(ByteBuffer.wrap(bytes));
            outputChannel.transferFrom(inputChannel, offset + bytes.length, orgSize - bytes.length);

The added lines should be removed.